### PR TITLE
TLauncher

### PR DESCRIPTION
**Pull request by jurf**
[TLauncher](https://tlauncher.org/en/) is an alternative Minecraft launcher for easier managing of Minecraft versions, mods and skins.

Based on the Minecraft package.

## TODO:
- [x] Update screenshots
- [x] Update OARS

There’s also the question of whether this belongs on Flathub, as it allows to run Minecraft without an official account. On the other side, Google Play doesn’t seem to have a problem with this.

#QuarantineCulture

### CODE REVIEW

**Comment by jurf** [comment]
bot, build org.tlauncher.TLauncher

**Comment by jurf** [comment]
bot, build org.tlauncher.TLauncher

**Comment by jurf** [comment]
bot, build org.tlauncher.TLauncher

File: org.tlauncher.TLauncher.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 TLauncher Inc. -->
+<component type="desktop-application">
+  <id>org.tlauncher.TLauncher</id>
+  <name>TLauncher</name>
+  <summary>The launcher for Minecraft</summary>
+  <developer_name>TLauncher Inc.</developer_name>
+
+  <description>
+    <p>Today not many good launcher for the game Minecraft, but they are
```
Comment (barthalion) [inline org.tlauncher.TLauncher.metainfo.xml]: There are grammar error here and there.

File: org.tlauncher.TLauncher.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 TLauncher Inc. -->
+<component type="desktop-application">
+  <id>org.tlauncher.TLauncher</id>
+  <name>TLauncher</name>
+  <summary>The launcher for Minecraft</summary>
```
Comment (barthalion) [inline org.tlauncher.TLauncher.metainfo.xml]: Summary could be better.

File: org.tlauncher.TLauncher.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  <developer_name>TLauncher Inc.</developer_name>
+
+  <description>
+    <p>Today not many good launcher for the game Minecraft, but they are
+    still there, and one of them TLauncher.</p>
+
+    <p><em>All official versions, from Alpha to Snapshots:</em>The launcher
```
Comment (barthalion) [inline org.tlauncher.TLauncher.metainfo.xml]: Currently anything within `<em>` tags ends up as `(null)` on the website. You may want to remove it.

File: org.tlauncher.TLauncher.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+modules:
+  # Needed by Minecraft 1.8.2 and up
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz
+        sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+
+  - name: gamemode
```
Comment (barthalion) [inline org.tlauncher.TLauncher.yaml]: I'm not sure but isn't it part of the runtime?

**Comment by barthalion** [comment]
Have you tried to reach out to upstream?

File: org.tlauncher.TLauncher.metainfo.xml (hunk 1 above)
Comment (jurf) [reply org.tlauncher.TLauncher.metainfo.xml]: Cut and paste from the site, I’d need to do a rewrite to fix all the errors, so I just let it be.

File: org.tlauncher.TLauncher.metainfo.xml (hunk 3 above)
Comment (jurf) [reply org.tlauncher.TLauncher.metainfo.xml]: Ah well.

File: org.tlauncher.TLauncher.yaml (hunk 4 above)
Comment (jurf) [reply org.tlauncher.TLauncher.yaml]: Not that I know of, Steam bundles it. Would be a good idea to include it now that we have a portal, though.

**Comment by jurf** [comment]
I have not, I want to shoot them an email once I have an app site to showcase.

File: org.tlauncher.TLauncher.metainfo.xml (hunk 2 above)
Comment (jurf) [reply org.tlauncher.TLauncher.metainfo.xml]: Not sure how to make it better. The Twitter page states "The popular Minecraft launcher". I guess a compromise would be "A popular launcher for Minecraft"? Not sure what else to put inside.

**Comment by LeandroStanger** [comment]
bot, build org.tlauncher.TLauncher

**Comment by LeandroStanger** [comment]
bot, build org.tlauncher.TLauncher

File: org.tlauncher.TLauncher.metainfo.xml (hunk 2 above)
Comment (barthalion) [reply org.tlauncher.TLauncher.metainfo.xml]: Whatever to indicate it's not official launcher.

**Comment by barthalion** [comment]
> I have not, I want to shoot them an email once I have an app site to showcase.

Please do it before the application is merged.

**Comment by jurf** [comment]
Email sent.

**Comment by jurf** [comment]
It’s been 5 days, I didn’t get a reply. I’ll try to contact them again later, but the maintenance of this thing is minimal, so if it’s okay with you, requesting merge. :-)

**Comment by LeandroStanger** [comment]
bot, build org.tlauncher.TLauncher

**Pull request closed without merging**